src/Settings.js

```diff
@@ -95,29 +91,6 @@ export class Settings {
         }
     }
 
-    /**
-     * @returns {Promise<boolean|null>}
-     */
-    async getIncontextSignupInitiallyDismissed () {
```

Comment:
This verbose pattern of fetching and verifying the data was for the global configuration, feature toggles and available input types since they affect ALL features one way or another.

I can see why you added these here, and we probably need to update the docs/guides to be clearer, but we shouldn't be expanding `Settings` to include any new state from individual features. :)